_posts/en/newsletters/2023-02-15-newsletter.md

```diff
@@ -0,0 +1,259 @@
+      transferred in Bitcoin transactions.  This would have several
+      advantages:
+
+      - *Reduced costs:* no transaction fees need to be paid for data
+        stored offchain.
+
+      - *Private:* two people can exchange a colored coin without
+        anyone else knowing anything about the data it references.
+
+      - *No transaction required for creation:* data can be associated
+        with an existing UTXO; there's no need to create a new UTXO.
+
+      - *Censorship resistant:* if the association between the data
+        and the colored coin is not widely known, then transfers of
+        the colored coin are just as censorship resistant as any other
+        onchain Bitcoin payment.
+
+      Considering the censorship resistant aspect, Towns argues that
+      "coloured bitcoins is largely unavoidable and simply something that
+      must be dealt with, rather than something we should spend time
+      trying to prevent/avoid."  He compares the idea that colored coins
+      might have more value than fungible bitcoins to the operation of
+      Bitcoin charging transaction fees based on transaction weight
+      rather than value transferred, concluding that he doesn't believe
+      this necessarily leads to significantly misaligned incentives.
+
+    - *Increasing allowed `OP_RETURN` space in standard transactions:*
+      Christopher Allen [asked][allen op_return] whether it was better
+      to put arbitrary data in a transaction output using `OP_RETURN` or
+      the witness data of a transaction.  After some discussion, several
+      participants ([1][todd or], [2][o'connor or], [3][poelstra or])
+      noted that they were in favor of relaxing default transaction
+      relay and mining policies to allow `OP_RETURN` outputs to store
+      more than 83 bytes of arbitrary data.  They reasoned that other
+      methods for storing large amounts of data are currently in use and
+      there would be no additional harm from `OP_RETURN` being used
+      instead.
+
+- **Fee dilution in multiparty protocols:** Yuval Kogman
+  [posted][kogman dilution] to the Bitcoin-Dev mailing list the
+  description of an attack against certain multiparty protocols.
+  Although the attack was [previously described][riard dilution],
+  Kogman's post brought renewed attention to it.  Imagine that Mallory
+  and Bob each contribute one input to a joint transaction with an
+  expected size and fee, implying an expected feerate.  Bob provides a
+  witness of the expected size for his input but Mallory provides a much
+  larger witness than expected.  This effectively decreases the feerate
+  for the transaction.  Several implications of this were discussed on
+  the mailing list:
+
+    - *Mallory gets Bob to pay her fees:* if Mallory has some ulterior
+      motive for including a large witness in the block chain---for
+      example, she wants to add arbitrary data---she can get Bob to pay
```

Comment:
This part wasn't clear to me, even after reading the ML post. How does she "get Bob to pay" the fees for her large input? He's already contributed an input whose value (the value of the UTXO it references) presumably covers his portion of the required fees (his input plus his share of the outputs and fixed part of the tx). How does this "effectively steals a fee from Alice" (quoting from the ML, but here we're saying Bob instead of Alice)? What exactly happens with this attack, does Bob see that Mallory has created this large input without providing enough fee to cover it, so he says, well, I better contribute a larger input to cover this fee? I don't think so, this isn't multiple rounds.

But if this is just me not understanding, ignore this comment. Otherwise, maybe consider making this a bit more explicit, if doing so doesn't add too much verbiage.

Reply:
Added a few extra words to the sentence to avoid "gets" and added a two sentence example to clarify what's happening (I hope).